apps/app/src/client/components/ReactMarkdownComponents/Header.tsx

```diff
@@ -111,7 +113,7 @@ export const Header = (props: HeaderProps): JSX.Element => {
     };
   }, [activateByHash, router.events]);
 
-  const showEditButton = !isGuestUser && !isReadOnlyUser && !isSharedUser && shareLinkId == null;
+  const showEditButton = !isGuestUser && !isReadOnlyUser && !isSharedUser && shareLinkId == null && currentPageYjsData?.hasRevisionBodyDiff === false;
```

Comment:
以下の主旨をコメントで残してください

> Yjs のドラフトがある場合は Header 横の編集ボタンを表示しない。これはドラフトがもし空文字だった場合、現状の条件分岐では対応できないためであり、 ySyncAnnotation を import できるようになったときに対応が可能になる。

Reply:
コメントしました。
```
  // TODO: currentPageYjsData?.hasRevisionBodyDiff === false make to hide the edit button when a Yjs draft exists
  // This is because the current conditional logic cannot handle cases where the draft is an empty string.
  // It will be possible to address this TODO ySyncAnnotation become available for import.
  // Ref: https://github.com/yjs/y-codemirror.next/pull/3
```